_episodes/06-first_session_end.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "End of first session"
+teaching: 0
+exercises: 0
+questions:
+- "Have you marked your attendance for this first session?"
+objectives:
+- "Please complete the attendance for this first session."
+keypoints:
+- Complete the attendance for this first session.
```

Comment:
Ah the things we have to do to keep linters happy :laughing:

Reply:
@alexdewar it reminded me of the English grammar exercises to transform a given sentence into declarative, interrogative, etc. 😄